packages/jaeger-ui/src/components/common/NameSelector.test.js

```diff
@@ -14,22 +14,18 @@
 
 import React from 'react';
 import { render, screen, waitFor, within, cleanup } from '@testing-library/react';
-import userEvent from '@testing-library/user-event'; // Use userEvent for more realistic interactions
-import '@testing-library/jest-dom'; // For custom matchers
+import userEvent from '@testing-library/user-event';
+import '@testing-library/jest-dom';
 
 import NameSelector, { DEFAULT_PLACEHOLDER } from './NameSelector';
 
-// Mock FilteredList as its internal behavior is tested separately
```

Comment:
once again, why delete the comments? When making changes to existing codebase the objective is always to make as few changes as possible, only when necessary, or to improve the code (deleting comments does not improve code)